components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -2505,63 +2505,65 @@ public static boolean isAnalyticsEnabled() {
         return propertyMap;
     }
 
+	public static void addDefinedAllSequencesToRegistry(UserRegistry registry,
+	                                                    String customSequenceType)
+			throws APIManagementException {
+
+		try {
+			File inSequenceDir = new File(
+					APIConstants.API_CUSTOM_SEQUENCES_FOLDER_LOCATION + File.separator +
+					customSequenceType);
+			File[] sequences = inSequenceDir.listFiles();
+
+			for (File sequenceFile : sequences) {
+				String sequenceFileName = sequenceFile.getName();
+				String regResourcePath =
+						APIConstants.API_CUSTOM_SEQUENCE_LOCATION + File.separator +
+						customSequenceType + File.separator + sequenceFileName;
```

Comment:
"regResourcePath" is a registry path and it's not a file path. File.separator returns different characters for different OS (Eg: windows = \ , unix = /). Thus different OS may produce different "regResourcePath" values. Do we need different registry paths based on OS?
And also APIConstants.API_CUSTOM_SEQUENCE_LOCATION = APIMGT_REGISTRY_LOCATION+"/customsequences" and we have hard coded "/" in API_CUSTOM_SEQUENCE_LOCATION constant already. But here we have used File.separator to concat "customSequenceType" and "sequenceFileName" into APIConstants.API_CUSTOM_SEQUENCE_LOCATION